go/ct/spc/specification.go

```diff
@@ -908,7 +908,9 @@ func getAllRules() []Rule {
 		effect: func(s *st.State) {
 			targetBlockNumber := s.Stack.Pop()
 			index := s.BlockContext.BlockNumber - targetBlockNumber.Uint64()
-			s.Stack.Push(NewU256FromBytes(s.RecentBlockHashes[index-1][:]...))
+			// we don't need to check for index ∈ [0..255] because InRange256FromCurrentBlock already does that
```

Comment:
until we change the implementation and we silently break this code.
no catch all, no ignore err, plz

Reply:
As discussed offline, CT does not run in client and cannot produce failure there. It is kind of ok to ignore it here.